tests/test_activation.py

```diff
@@ -28,13 +28,14 @@
 import unittest
 from unittest.mock import Mock, patch
 
+from csm_api_client.service.gateway import APIError
+
 from cfs_config_util.activation import (
     cfs_activate_version,
     cfs_deactivate_version,
     ensure_product_layer
 )
-from cfs_config_util.apiclient import APIError
-from cfs_config_util.cfs import (
+from csm_api_client.service.cfs import (
```

Comment:
Should group this import with the one on line 31 now instead of with imports from `cfs_config_util`.